skins/oasis/js/Tracking.js

```diff
@@ -583,12 +536,11 @@ jQuery(function($){
 				return;
 			}
 
-			switch(id) {
-				case 'wall-notifications-markasread-this-wiki':
-				case 'wall-notifications-markasread-all-wikis': {
-					label = id;
-					break;
-				}
+			switch (id) {
+			case 'wall-notifications-markasread-this-wiki':
```

Comment:
Created https://github.com/Wikia/guidelines/pull/58 to handle the discrepancy. The code is correct, the guidelines were wrong. Thanks for catching!